special-pages/messages/new-tab/initialSetup.response.json

```diff
@@ -25,6 +25,16 @@
           "enum": ["macos", "windows", "android", "ios", "integration"]
         }
       }
+    },
+    "updateNotification": {
+      "oneOf": [
+        {
+          "type": "null"
+        },
+        {
+          "$ref": "types/update-notification.json"
+        }
+      ]
     }
   }
```

Comment:
nullable in the `initialSetup` message to prevent macOS needing to give dummy values